packages/storage/src/tasksnapshot.ts

```diff
@@ -24,7 +24,13 @@ import { UploadTask } from './task';
  * @public
  */
 export interface UploadResult {
+  /**
+   * Contains the metadata sent back from the server.
+   */
   readonly metadata: Metadata;
+  /**
```

Comment:
This file is also missing some empty lines. You can push back if you want :)

Reply:
I was going to say if we are going to enforce it we should probably set Prettier to do it but it seems like it can't.